packages/messages/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "message",
+  "version": "0.0.0",
+  "description": "This package contains all business rules for messaging system",
+  "author": "Hector Esau Macias Javier <hecttoressau@gmail.com>",
+  "homepage": "https://github.com/Platzi-Master-C9/backend-services/tree/master/packages/message#readme",
+  "license": "ISC",
+  "main": "src/index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Platzi-Master-C9/backend-services.git"
+  },
+  "scripts": {
+    "test": "jest"
+  },
```

Comment:
Can we add 2 extra scripts for eslint? It could something like this:
```json
{
  "scripts": {
    ...other scripts,
     "lint": "eslint .",
     "lint:fix": "eslint . --fix"
  }
}
```
If we add these scripts I recommend to add the `.eslintignore` file with the following content:
```
jest.config.js
node_modules
```